great-idea/services.html

```diff
@@ -16,91 +16,114 @@
-eleifend. Mauris euismod facilisis iaculis.
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-Learn More
+<button>Learn More</button>
+</div>
+</section>
 
+<section class="middle-contentsr">
+  <div class="digmarket-text">
+<h5>Digital Marketing</h5>
 
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-iOS Development
+<button>Learn More</button>
+</div>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+  <div class="webdev-text">
+  <h5>Web Development</h5>
 
-Learn More
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
+<button>Learn More</button>
+</div>
+</section>
 
+<section class="lower-contentsr">
+  <div class="iosdev-text">
+<h5>iOS Development</h5>
 
-Android Development
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<button>Learn More</button>
+  </div>
 
-Learn More
+  <div class="androiddev-text">
+<h5>Android Development</h5>
 
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-Some Facts About Our Services
+<button>Learn More</button>
+</div>
+</section>
 
```

Comment:
So... I'm noticing that you're using a lot of separate classes for your box content here. However, since the styling is the same for most of the content boxes, you could simplify and [DRY](https://g.co/kgs/7NNBxH) your code by using one class for each box.

e.g.
```html
<section class="card-content">
  <div class="card">
    <h5>iOS Development</h5>
    <p>Paragraph of text...</p>
    <button>Learn More</button>
  </div>
  <div class="card">
    <h5>Android Development</h5>
    <p>Paragraph of text...</p>
    <button>Learn More</button>
  </div>
</div>
```